messages/help.txt

```diff
@@ -1,7 +1,10 @@
-*ŠALÁT + MÄSO*
-stačí nahlásiť do ~11:00, že budete papať a dostanete šalátik + mäsko, aké si vypýtate z Billy. Bežne býva kuracie stehno, klobáska, no nájde sa občas aj sekaná, prsia, bôčik, ...
+*STÁLA PONUKA*
+Každý deň sa v kuchyni pripravujú pečené kuracie stehná (asi 3kg) a dusená zelenina.
```

Comment:
Ok. But there is no agreed plan to appropriately respond to these "buy chicken" requests. From what I have found, at the moment, ~3kgs of chicken legs are baked every day no matter what. How many people should explicitly request it, in order for more legs to be prepared? 3, 10, 15? It is unclear and as a result, no correction is done. At least some kind of rule around this would be nice.